samples/snippets/requirements.txt

```
@@ -29,6 +29,6 @@ requests==2.27.1
 rsa==4.8
 shapely==1.8.0
 six==1.16.0
-sqlalchemy==1.4.26
+sqlalchemy==1.4.29
```

SQLAlchemy 1.4.27 is known not to be working. https://github.com/googleapis/python-bigquery-sqlalchemy/issues/385

```suggestion
sqlalchemy==1.4.26
```